SESSION_FACILITATOR_GUIDE.md

```diff
@@ -0,0 +1,31 @@
+# 💁🏽 Session Facilitator Guide
+
+This document is intended to provide a helpful starting point or reference for running sessions during summits.
+The new learnings and tools from a summit will ideally be PR'd in when each is concluded.
+
+🎉 Lists are included 🎉
+
+## 📅 in the days leading up to the session
+- [ ] If applicable, port the agenda into [tcq](https://tcq.app/)
+- [ ] Set a chair/facilitator for the session
+
+## 🛫 pre-flight checklist (when about to begin the session)
+- [ ]  If not pre-arranged, find consensus on a chair for the session. Scheduled sessions are likely to already have a facilitator/chair. Ad-hoc sessions will need someone to make sure everyone walks out of the room with actionable items.
+- [ ] If not pre-arranged, ask the table for volunteer note taker.
+- [ ] Have a person be take charge of each tool being used for the session. For a full list of tools see below.
+- [ ] Go to Zoom, start the call.
+- [ ] In Zoom, make sure the link to the zoom session is posted on the relevant thread in [nodejs/summit](https://github.com/nodejs/summit/issues/).
+- [ ] In Zoom, start recording the session.
+- [ ] Go to Otter, start recording the audio.
```

Comment:
Otter?

Reply:
It's a tool to record speech/meeting audio. It's reasonable at transcribing the speech of multiple people. 

The audio can be exported as an .mp3

The auto transcribed text (of reasonable accuracy) can be exported as text.

Reply:
Point understood. Making it optional; I've used it, but we don't need something new for people to learn/remember on the day of.